biomaj/bank.py

```diff
@@ -525,10 +527,36 @@ def save_session(self):
 
         self.bank = self.banks.find_one({'name': self.name})
 
+    def format_release(self, release, reverse=False):
+        """
+        Check the release does not contains any dot '.' character that should make MongoDB complain
+
+        :param release: release number to check
+        :type release: str
+        :param reverse: Do the reverse replacement
+        :type reverse: bool
+        :return: Release string with dot '.' replaced with character set in config (release.replace)
+        :rtype: str
+        """
+        if not self.config.get('release.replacedot'):
```

Comment:
Yeap it could be the single place. However, if we fix this place, we could impact all bank that have this dot in their release value and so, we will have to rebuild/download each bank as the stored release value will be then different/changed.